.github/everything.Dockerfile

```diff
@@ -33,10 +33,7 @@ RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86
     mkdir -p /workspace/data && \
     chown -R gitpod:gitpod /opt/conda /workspace/data
 
-ENV PATH="/opt/conda/bin:$PATH"
-
-# Add classify to path
-ADD bin/classify /usr/local/bin/classify
```

Comment:
I'd rather we add bin than classify, it's more nextflowy, and future proof

Reply:
Fair. I was thinking of keeping the option open to allow for minor changes to the classify exec in the container, but have since changed my mind. I'll rebuild and push the image.